util/cli/cli_test.go

```diff
@@ -0,0 +1,29 @@
+package cli
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPromptPassword_Fallback(t *testing.T) {
+	oldStdin := os.Stdin
+	defer func() {
+		os.Stdin = oldStdin
+	}()
+
+	r, w, err := os.Pipe()
+	if err != nil {
+		t.Fatalf("Failed to create pipe: %v", err)
+	}
+	_, err = w.WriteString("test-password\n")
+	if err != nil {
+		t.Fatalf("Failed to write to pipe: %v", err)
+	}
+	w.Close()
+
+	os.Stdin = r
+	password := PromptPassword("")
+	require.Equal(t, "test-password", password)
```

Comment:
I personally would prefer to have a const for `test-password` that gets used both for a `WriteString` and the `Equal()` that makes it easier for someone to understand that the input and output are related and to change the value if they have a reason to.

Reply:
good suggestion, marking it as resolved.